requirements.txt

```diff
@@ -1,5 +1,5 @@
 git+https://github.com/ubccr/coldfront@2e60db247fd9a9b0d1299f1e98f1e4ef5f5c259b#egg=coldfront
-git+https://github.com/nerc-project/coldfront-plugin-openstack@15a524fc960e7d00828af774a07ee49b7dd6066d#egg=coldfront_plugin_openstack
+git+https://github.com/nerc-project/coldfront-plugin-cloud@15a524fc960e7d00828af774a07ee49b7dd6066d#egg=coldfront_plugin_openstack
```

Comment:
Gimme one sec to update the module name as well. (on `#egg=`)

Reply:
Sure no worries - I think I have all the references updated here but we'll need a new hash once the changes are in on the `coldfront-plugin-cloud` side.

Reply:
Updated to latest `coldfront-plugin-cloud` hash `9b3b34bc917efe1831a5ef5b7cb76a5ab85d297d`